packages/nc-gui/components/project/spreadsheet/rowsXcDataTable.vue

```diff
@@ -1246,6 +1248,8 @@ export default {
           this.expandRow(data.length - 1, rowMeta)
         }
       }
+      await this.save()
+      this.syncCount()
```

Comment:
Based on the previous comment, you don't need to call `syncCount()` in `insertNewRow()` because you should only call it upon a _successful_ insertion, i.e. somewhere in `save()`.

Please delete this line and add it in `save()`.